From 821a87411fd75bc1137e99250689f21d820e49ed Mon Sep 17 00:00:00 2001
From: Dominik Csapak <d.csapak@proxmox.com>
Date: Tue, 16 May 2017 10:36:15 +0200
Subject: [PATCH 10/10] use only app.js

because we will use the commonjs version,
we only need to use the generated app.js

Signed-off-by: Dominik Csapak <d.csapak@proxmox.com>
---
 vnc.html | 17 +----------------
 1 file changed, 1 insertion(+), 16 deletions(-)

diff --git a/vnc.html b/vnc.html
index a139e26..46d09d7 100644
--- a/vnc.html
+++ b/vnc.html
@@ -46,22 +46,7 @@
     </script>
 
     <!-- begin scripts -->
-    <!-- promise polyfills promises for IE11 -->
-    <script src="/novnc/vendor/promise.js"></script>
-    <!-- ES2015/ES6 modules polyfill -->
-    <script type="module">
-        window._noVNC_has_module_support = true;
-    </script>
-    <script>
-        window.addEventListener("load", function() {
-            if (window._noVNC_has_module_support) return;
-            var loader = document.createElement("script");
-            loader.src = "/novnc/vendor/browser-es-module-loader/dist/browser-es-module-loader.js";
-            document.head.appendChild(loader);
-        });
-    </script>
-    <!-- actual script modules -->
-    <script type="module" crossorigin="anonymous" src="/novnc/app/ui.js"></script>
+    <script crossorigin=anonymous" src="/novnc/app.js"></script>
     <!-- end scripts -->
 </head>
 
-- 
2.11.0

